services/state/roster/roster_entry.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
```

Comment:
It wouldn't hurt to include a file comment here, but it's not strictly required at the moment.